sklearn/metrics/tests/test_pairwise.py

```diff
@@ -78,7 +79,7 @@ def test_pairwise_distances():
     S3 = manhattan_distances(X, Y, size_threshold=10)
     assert_array_almost_equal(S, S3)
     # Test cosine as a string metric versus cosine callable
-    # "cosine" uses sklearn metric, cosine (function) is scipy.spatial
+    # "cosine" uses scikit-learn metric, cosine (function) is scipy.spatial
```

Comment:
I'm on the fence about whether this should be changed, because it implicitly refers to `sklearn.metrics`. It'd probably fine as it is / the meaning is clear regardless which name is used